<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClaimDetail</class>
 <widget class="QDialog" name="ClaimDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер рекламации</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата поступления</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>От кого получено</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип рекламации</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Эксплуатирующая организация</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Объект установки</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Краткое описание</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип комплектующих</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ВНФТ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Количество</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип оборудования</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Зав. № модуля</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Зав. № модуля для замены</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>№ оборудования</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>70</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата отгрузки</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>100</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Гарантия</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Вероятная причина</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>160</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата устранения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>190</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Меры устранения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>230</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Файл ответа</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>260</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Заключение</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNumber">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leFromWho">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="deDateClaim">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cbTypeClaim">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leOrganiz">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leObjectInst">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDescript">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbTypeComplect">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leVNFT">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbQuantity">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QComboBox" name="cbTypeDevice">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNumModul">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>360</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNumNewModul">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>400</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNumDevice">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>30</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="deDateOut">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>60</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbGuarantee">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>90</y>
     <width>150</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="leReason">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>120</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="deDateRepair">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>150</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDoRepair">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>180</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leFileAnswer">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>210</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leTextResult">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>250</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOK">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>400</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>400</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>ClaimDetail</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>639</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
